internal/sdkv2provider/schema_cloudflare_dlp_profile.go

```diff
@@ -142,5 +142,10 @@ func resourceCloudflareDLPProfileSchema() map[string]*schema.Schema {
 				Schema: resourceCloudflareDLPContextAwarenessSchema(),
 			},
 		},
+		"ocr_enabled": {
+			Type:        schema.TypeBool,
+			Required:    true,
```

Comment:
is this intentional? this will break existing implementations given they won't have this in their resources. can it be optional? or with a default if we much supply a value?

Reply:
with this marked as optional, the tests are passing. being required, it breaks the test suite.

Reply:
fixed, sorry